src/containers/Routes/components/Drawer/Drawer.tsx

```diff
@@ -36,6 +37,6 @@ export const DrawerMenu = () => {
           </Fragment>
         );
       })}
-    </WrapperContainer>
+    </SafeAreaView>
```

Comment:
Try to implement logic of opening mail and calling to phone using Linking
https://reactnative.dev/docs/0.68/linking
<img width="189" alt="image" src="https://user-images.githubusercontent.com/36453308/208439397-06442149-ff78-4f57-8be6-478264c7e1b5.png">

Reply:
done